front_end/html/edit_assignment.html

```diff
@@ -247,6 +247,25 @@ <h2>Create assignment</h2>
         </p>
     </div>
 
+    <div class="shadow-box">
+        <p><strong>Restrict assignment access to the following IP addresses: </strong></br>Separate valid addresses by a comma without spaces.</p>
+        <div class="select is-grey">
+            <select id="access_restricted" onchange="showAllowedIPAddresses(this.value)" name="access_restricted" class="edit-select">
+                {% if assignment_details["allowed_ip_addresses"] is not None %}
```

Comment:
Again, this is nitpicky, but please simplify this to:

`if assignment_details["allowed_ip_addresses"]`

It does the same thing (if only returns true if the value is not None).